manifests/crds/analysis-run-crd.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.2.5
+    controller-gen.kubebuilder.io/version: (devel)
```

Comment:
Does this need to be a specific version or is `(devel)` replaced before a release?

Reply:
I this so but I'm not sure how to